README.md

```diff
@@ -1,3 +1,36 @@
+S3L1M created a very useful program to help revert files captured by `ground.exe` virus.  I got infected, amd noticed it fairly quickly when I noticed `ground.exe` running in task manager.  I would like to create this updated readme with more detailed instructions for using Ground-Damage-Fixer. Of course, you already have `Ground` stopped and are stuck with this messy aftermath. I included how I removed `Ground.exe` below in case you came across this page while still searching for a way to remove the virus.
```

Comment:
What you wrote here should be in the description of pull request not in the readme itself